.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: ruby
+rvm:
+ - 2.0.0
+ - 2.1.7
+ - 2.2.3
```

Comment:
We don't need to define patch version, let's define 2.0, 2.1 and 2.2. It will build in latest.